### PR TITLE
Doc 92 - Critical Assets Article

### PR DESCRIPTION
**Pull request by janettelynch**
@a-u-h-g Here is a first draft of an article that groups all the previously disparate info about critical assets. Let me know if there are any errors or other info you would like me to add. Thanks.

### CODE REVIEW

File: knowledgeBase/asset-management/critical-assets.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Understanding Critical Assets
+
+Critical assets is a class of asset for which you can create queries and alerts to quickly access the most crucial data. By default, JupiterOne determines which criteria defines an asset as the most important and, therefore, the most at risk but an administrator can edit this definition.
+
+## Quick Filtering the Critical Assets
+
+In J1 Assets, there are two tabs: All Assets and Critical Assets. Click the **Critical Assets** tab at the top of the page to go directly to the most important of your assets.
+
+
+![](C:\Users\lynch\OneDrive\Documents\GitHub\docs\knowledgeBase\assets\assets-landing.png)
```
Comment (a-u-h-g) [inline knowledgeBase/asset-management/critical-assets.md:10]: these image addresses should be pointing to the relative file path in the github repo - currently pointing to your local machine's hard drive

File: knowledgeBase/asset-management/critical-assets.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+
+Click ![](C:\Users\lynch\OneDrive\Documents\GitHub\docs\knowledgeBase\assets\icons\gear.png) to edit the critical asset definition default values. You can use classes, properties, and values to define what is critical.
```
Comment (a-u-h-g) [inline knowledgeBase/asset-management/critical-assets.md:14]: these image addresses should be pointing to the relative file path in the github repo - currently pointing to your local machine's hard drive

File: knowledgeBase/asset-management/critical-assets.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+![](C:\Users\lynch\OneDrive\Documents\GitHub\docs\knowledgeBase\assets\asset-definition.png)
```
Comment (a-u-h-g) [inline knowledgeBase/asset-management/critical-assets.md:16]: these image addresses should be pointing to the relative file path in the github repo - currently pointing to your local machine's hard drive

File: knowledgeBase/asset-management/critical-assets.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+![](C:\Users\lynch\OneDrive\Documents\GitHub\docs\knowledgeBase\assets\asset-definition.png) 
+
+Add asset classes and properties that your organization considers a critical asset, and click **Update Definition**. Default critical asset classes include `Application`, `CodeRepo`, `Datastore`, `Function`, `Host`, and `Logs`.
+
+## Query Critical Assets
+
+To know which critical assets related to compliance gaps may need remediation, you can run queries such as:  `find #CriticalAsset that has jupiterone_compliance_gap` as shown in this example:
+
+
+![0o](C:\Users\lynch\OneDrive\Documents\GitHub\docs\knowledgeBase\assets\problems-query.png)
```
Comment (a-u-h-g) [inline knowledgeBase/asset-management/critical-assets.md:25]: this one has a subheading of '0o'? also these image addresses should be pointing to the relative file path in the github repo - currently pointing to your local machine's hard drive

**Review by a-u-h-g — CHANGES_REQUESTED** [review]

File: knowledgeBase/asset-management/critical-assets.md (hunk 2 above)
Comment (a-u-h-g) [reply knowledgeBase/asset-management/critical-assets.md:14]: this one still needs to be fixed I think

File: knowledgeBase/asset-management/critical-assets.md (hunk 4 above)
Comment (a-u-h-g) [reply knowledgeBase/asset-management/critical-assets.md]: Do we want '0o' as the mouseover text for some reason?

**Review by a-u-h-g — APPROVED** [review]

**Review by sarabeth — APPROVED** [review]

**Pull request merged**